tlog.spec

```diff
@@ -2,9 +2,9 @@
 
 %if 0%{?rhel} == 0 && 0%{?rhel} < 7
 # If it's RHEL6 and older
-%bcond_with systemd
-%else
 %bcond_without systemd
+%else
+%bcond_with systemd
```

Comment:
This inverts the logic and makes it so that `systemd` dependencies are enabled on RHEL 6 and older while being disabled everywhere else.